xml/compute.xml

```diff
@@ -820,215 +426,94 @@
    <para>
     OpenBLAS is an optimized BLAS (Basic Linear Algebra Subprograms) library
     based on GotoBLAS2 1.3, BSD version. It provides the BLAS API. It is
```

Comment:
From here until the trilinos section, the diff gets a little weird (thank you git). What actually happened is simple though: petsc, scalapack, and scotch were _removed_, and openblas is _still intact_.